client/src/components/Login.style.js

```diff
@@ -0,0 +1,85 @@
+import styled from 'styled-components';
+
+export const StyleAll = styled.div`
+  background-color: #f9d52f;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-flow: column wrap;
+`;
+
+export const Title = styled.h1`
+  fontfamily: Work Sans;
+  fontstyle: Black;
+  fontsize: 38px;
+  background: white;
+  margin: 50px;
+`;
+export const StyledForm = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-flow: column wrap;
+`;
+export const InputContainer = styled.div``;
+export const StyledLabel = styled.label`
+  font-size: 18px;
+  text-align: center;
+  color: black;
+  font-weight: bold;
+`;
+export const StyledText = styled.h1`
+  font-size: 18px;
+  text-align: center;
+  color: black;
+`;
+export const StyledInput = styled.input`
+  display: block;
+  background: #ffffff;
+  border-radius: 10px;
+  margin-bottom: 30px;
+  width: 200px;
+  padding: 10px 0px;
+`;
+export const StyledBottom = styled.div``;
+
+export const LoginButton = styled.button`
+  width: 168px;
+  height: 32px;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  border-radius: 20px;
+  background-color: #1d1a1ae0;
+  font-family: Roboto;
+  font-style: normal;
+  font-weight: 900;
+  font-size: 20px;
+  line-height: 23px;
+  letter-spacing: 0.08em;
+  color: white;
+`;
+export const StyledP = styled.h1`
+  font-family: Roboto;
+  font-style: normal;
+  font-weight: 900;
+  font-size: 11px;
+  text-decoration-line: underline;
+  margin: 50px;
+`;
+export const Styleques = styled.img`
```

Comment:
```Styleques Stylephoto Stylecontainer``` can all be removed since they will be replace by the actual header component :)

Reply:
Ok